internal/moduleconfig/moduleconfig.go

```diff
@@ -178,17 +202,28 @@ func setConfigDefaults(moduleDir string, config *ModuleConfig) error {
 
 	// Do some validation.
 	if !isBeneath(moduleDir, config.DeployDir) {
-		return fmt.Errorf("deploy-dir must be relative to the module directory")
+		return fmt.Errorf("deploy-dir %s must be relative to the module directory %s", config.DeployDir, moduleDir)
 	}
 	for _, deploy := range config.Deploy {
 		if !isBeneath(moduleDir, deploy) {
-			return fmt.Errorf("deploy files must be relative to the module directory")
+			return fmt.Errorf("deploy %s files must be relative to the module directory %s", deploy, moduleDir)
 		}
 	}
 
 	return nil
 }
 
+func fileExists(filename string) bool {
+	_, err := os.Stat(filename)
+	if err == nil {
+		return true
+	}
+	if os.IsNotExist(err) {
+		return false
+	}
+	return false
```

Comment:
This should be equivalent:
```suggestion
	return err == nil
```

Reply:
Oops, I may have just copied that method from copilot.